cloudbees-cd/kubernetes/logs/get-logs.sh

```diff
@@ -0,0 +1,93 @@
+#! /bin/bash
+
+
+kctl() {
+   if [ ! -z $KUBE_CONFIG_PATH ] ; then
+    kubectl --kubeconfig $KUBE_CONFIG_PATH  -n $NAMESPACE "$@"
+   else
+    kubectl -n $NAMESPACE "$@"
+   fi
+}
+
+function get_logs
+{
+  C_FETCH="flow"
+  if [ $COMPONENT == "all" ] ; then
+    C_FETCH="flow"
+  else
+    C_FETCH=$COMPONENT
+  fi
+  for i in $(kctl get po|awk '{print $1}'|awk 'NR!=1'|grep $C_FETCH)
+   do
+   echo "Fetching logs for $i"
+   echo "Creating Directory $OUTPUT/$i/logs"
+   mkdir -p $OUTPUT/$i/logs
+   echo "Copying logs from $i to  $OUTPUT/$i/logs"
+   kctl cp $i:/opt/cbflow/logs $OUTPUT/$i/logs
```

Comment:
Is this different from regular pod logs, for which you can just use `kubectl logs deploy/flow-server` or just `kail --current-ns` (IIRC)?

Reply:
@jglick  sometime people dont configure deploy namespace as current namespace so added wrapper for it